pkg/v20/chartconfig/error_internal.go

```diff
@@ -0,0 +1,21 @@
+package chartconfig
```

Comment:
Why `error_internal.go`?

Reply:
@xh3b4sd I'm trying to follow the same procedure we described in https://github.com/giantswarm/fmt/blob/master/go/errors.md#handling-third-party-library-errors

Reply:
I see, wasn't aware this is a thing now. Thanks for the explanation.

Reply:
This is misuse of the concept. As described in fmt matchers here should be private and not used outside the package. Their purpose is internal handling and errors "translation".

Reply:
This is true but then we need 4 copies of the matcher, 1 per controller.

I'm actually fine with that as its a consequence of the architecture we chose and we are cleaning it up. So long term there will only be the `clusterapi` controller.

Reply:
@rossf7 then this error can be "translated" and returned as this package error like in fmt example:

```
if isGitHub404(err) {
	return RepositoryFile{}, microerror.Maskf(notFoundError, "repository file %#q for owner %#q in repository %#q", path, owner, repo)
}
```

Reply:
Cool that works. I'll do that in the migration resource I'm adding. 

@tomahawk28 Can you do a follow up PR for the chartconfig resources?

Reply:
Thanks <3
This is especially important when we want to switch to `errors.Is` eventually.

Reply:
How can we make this more easily understandable? The concept does not seem to be clear and graspable for everyone yet.

Reply:
https://github.com/giantswarm/fmt/pull/76